openreview/conference/builder.py

```diff
@@ -1857,7 +1857,7 @@ def post_decision(paper_decision):
         if errors:
             error_status = f'''
 ```python
-{json.dumps(errors, indent=2)}
+{json.dumps({key: errors[key] for key in errors.keys()[:100]}, indent=2)}
```

Comment:
could you should the total of the errores somewhere? I would suggest showing only the first 10 and then tell the total of errors. The errors should be all the same.